service/src/main/java/org/apache/griffin/core/exception/GriffinExceptionMessage.java

```diff
@@ -72,7 +72,9 @@ Licensed to the Apache Software Foundation (ASF) under one
 
     HDFS_FILE_NOT_EXIST(40407, "Hadoop data file not exist"),
 
-    PREDICATE_TYPE_NOT_FOUND(40408, "Unknown predicate type"),
+    JOB_INSTANCE_NOT_FOUND(40408, "No job instances with given job instance id found"),
+
+    PREDICATE_TYPE_NOT_FOUND(40409, "Unknown predicate type"),
```

Comment:
This probably makes duplication with the next one. At least when requesting non-existing instance, I got `Unknown predicate type` in response :o

Reply:
Done.